website/versioned_docs/version-6.26.3/plugin-syntax-dynamic-import.md

```diff
@@ -5,6 +5,16 @@ sidebar_label: syntax-dynamic-import
 original_id: babel-plugin-syntax-dynamic-import
 ---
 
+`@babel/plugin-syntax-dynamic-import` is needed to enable parsing for import().
```

Comment:
This file contains the Babel 6 docs, so:
- packages start with `babel-`, not with `@babel/`
- We can remove the note about Babel 7.8.0